.yarnrc

```diff
@@ -0,0 +1 @@
+ignore-scripts: true
```

Comment:
This file will go away when we upgrade to Yarn v4, but is here just to get `yarn install` to work.